src/index.ts

```diff
@@ -128,6 +129,7 @@ export interface IDKeys {
 }
 
 export interface CreateOptions {
+  ttlNs?: bigint
```

Comment:
After some thought, I'd accept a number here to make the public API easier with just numbers, especially since you always convert to`BigInt` in the `create` function. I checked and the max int (Number.MAX_SAFE_INTEGER) is the equivalent of  2501 hours which is very unlikely to happen.

```suggestion
  ttlNs?: number
```